data/websites.json

```diff
@@ -1060,6 +1060,11 @@
       "url": "https://nitc.gov.np/",
       "description": "Official Website of Nepal Government's NITC"
     },
+    {
+          "name": "National Youth Council",
```

Comment:
Kindly use prettier to properly format the code and put the information in alphabetical order.

Reply:
Corrected the alphabetical order. LGTM now.